care/emr/resources/user/spec.py

```
@@ -155,6 +158,61 @@ def perform_extra_serialization(cls, mapping, obj: User):
+    date_of_birth: str | None
+    verified: bool
+    pf_endpoint: str | None
+    pf_p256dh: str | None
+    pf_auth: str | None
+    organizations: list[dict]
+    facilities: list[dict]
+    permissions: list[str]
+
+    @classmethod
+    def perform_extra_serialization(cls, mapping, obj: User) -> None:
+        from care.emr.resources.facility.spec import FacilityBareMinimumSpec
```

Ruff (0.12.2)

`import` should be at the top-level of a file

(PLC0415)